source/admin_manual/logging.rst

```diff
@@ -6,15 +6,45 @@ Dovecot Logging
 
 **Dovecot always logs a detailed error message** if something goes wrong. If it doesn't, it's considered a bug and will be fixed. However, almost always the problem is that **you're looking at the wrong log file**; error messages may be logged to a different file than informational messages.
 
+By default Dovecot logs to syslog using mail facility. Dovecot can also be configured to log to files directly.
+
+Log File Location
+^^^^^^^^^^^^^^^^^
+
 You can find the log file locations by running:
 
 .. code-block:: none
 
    doveadm log find
 
-Dovecot log configuration is found in the ``conf.d/10-logging.conf`` file in the dovecot configuration folder (usually ``/etc/dovecot`` but may also be ``/usr/local/etc/dovecot``).
+Last Errors
+^^^^^^^^^^^
+
+You can easily print the last 1000 error messages of a running Dovecot:
+
+.. code-block:: none
+
+   doveadm log errors
+
+Changing Log File Paths
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't want to use syslog, you can make Dovecot log to files directly:
 
-By default Dovecot logs to syslog using mail facility. You can change the facility from :ref:`setting-syslog_facility` setting. The syslog configuration is often in ``/etc/syslog.conf`` or ``/etc/rsyslog*`` files. You can also configure Dovecot to write to log files directly, see below.
+.. code-block:: none
+
+   log_path = /var/log/dovecot.log
+   # If you want everything in one file, just don't specify info_log_path and debug_log_path
+   info_log_path = /var/log/dovecot-info.log
+   # Leave empty in order to send debug-level messages to info_log_path
+   debug_log_path = /var/log/dovecot-debug.log
+
+The warning and error messages will go to file specified by :ref:`setting-log_path`, while informative messages goes to :ref:`setting-info_log_path` and debug messages goes to :ref:`setting-debug_log_path`. If you do this, make sure you're really looking at the :ref:`setting-log_path` file for error messages, since the "Starting up" message is written to :ref:`setting-info_log_path` file.
```

Comment:
Is the "If you do this..." sentence necessary? It feels a bit out of place. Maybe it makes sense. The tone of the whole thing is "weird" but that's probably just because this is a guide and not "normal" documentation :)

Reply:
This was some old existing text. But yeah, I think the beginning of the text already makes it clear enough how to find the error log. Removed the second sentence.